troposphere/codedeploy.py

```diff
@@ -80,11 +80,25 @@ class ElbInfoList(AWSProperty):
     }
 
 
+class TargetGroupInfoList(AWSProperty):
+    props = {
+        'Name': (basestring, False)
+    }
+
+
 class LoadBalancerInfo(AWSProperty):
     props = {
-        'ElbInfoList': ([ElbInfoList], False)
+        'ElbInfoList': ([ElbInfoList], False),
+        'TargetGroupInfoList': ([TargetGroupInfoList], False)
```

Comment:
I usually always add commas at the end of these lists to make future additions a 1 line change.

Reply:
Thank you, I'm always happy to learn best practices.